winch/codegen/src/isa/x64/asm.rs

```diff
@@ -689,6 +689,16 @@ impl Assembler {
         });
     }
 
+    pub fn popcnt(&mut self, src: Reg, size: OperandSize) {
+        assert!(self.isa_flags.has_popcnt(), "Requires has_popcnt flag");
+        self.emit(Inst::UnaryRmR {
+            size: size.into(),
+            op: args::UnaryRmROpcode::Popcnt,
+            src: Gpr::new(src.into()).unwrap().into(),
+            dst: Writable::from_reg(Gpr::new(src.into()).unwrap()),
```

Comment:
We currently have `impl From<Reg> for Gpr` and `impl From<Reg> for WritableGpr` so you could use those here if you wanted to reduce the boilerplate.